tests/src/Content/Text/BBCodeTest.php

```diff
@@ -390,6 +390,10 @@ public function dataExpandVideoLinks(): array
 				'expectedBBCode' => '[url=https://www.youtube.com/watch?v=hfwbmTzBFT0]https://www.youtube.com/watch?v=hfwbmTzBFT0[/url]',
 				'text'           => '[youtube]https://youtube.com/embed/hfwbmTzBFT0[/youtube]',
 			],
+			'task-14940-youtube-mobile' => [
+				'expectedBBCode' => '[url=https://m.youtube.com/watch?v=hfwbmTzBFT0]https://m.youtube.com/watch?v=hfwbmTzBFT0[/url]',
```

Comment:
```suggestion
				'expectedBBCode' => '[url=https://www.youtube.com/watch?v=hfwbmTzBFT0]https://ww.youtube.com/watch?v=hfwbmTzBFT0[/url]',
```

Your added coded exactly does that (like the other replaces) -> transforming to the www-endpoint of youtube. That's why the tests fail.